api/src/passage/management/commands/passage_zwaar_verkeer_hour_aggregation_v2.py

```diff
@@ -108,29 +96,20 @@ def _get_aggregation_query(self, run_date):
                  WHEN toegestane_maximum_massa_voertuig <= 50000 THEN 'klasse07_40001-50000'
                  WHEN toegestane_maximum_massa_voertuig <= 60000 THEN 'klasse08_50001-60000'
                  WHEN toegestane_maximum_massa_voertuig <= 70000 THEN 'klasse09_60001-70000'
-                 WHEN toegestane_maximum_massa_voertuig <= 80000 THEN 'klasse10_70001-80000'
-                 ELSE 'klasse11_80001'
+                 WHEN toegestane_maximum_massa_voertuig >  70000 THEN 'klasse10_70001'
+                 ELSE 'onbekend'
                END                                   AS
                toegestane_maximum_massa_voertuig,
 				voertuig_soort, 
 				inrichting,
 				europese_voertuigcategorie,
 				europese_voertuigcategorie_toevoeging,
-				versit_klasse,
--- 				brandstof_alcohol,
--- 				brandstof_benzine,
--- 				brandstof_cng,
--- 				brandstof_diesel,
--- 				brandstof_electriciteit,
--- 				brandstof_lng,
--- 				brandstof_lpg,
--- 				brandstof_niet_geregistreerd,
--- 				brandstof_waterstof,
--- 				meerdere_brandstoffen,
                 brandstoffen,
-				co2_uitstoot_gecombineerd,
-				co2_uitstoot_gewogen,
-				milieuklasse_eg_goedkeuring_zwaar,
+               CASE 
+                WHEN lengte <= 1000 THEN '01 0t/m1000'
```

Comment:
Volgens mij komt deze query niet van jou, maar ik check het ff. Is het de bedoeling om deze 0 te hebben?